modules/landing_zone/main.tf

```diff
@@ -22,29 +22,30 @@ resource "null_resource" "landing_zone_config" {
     command = "node ${path.module}/scripts/remove-config.js"
 
     environment = {
-      ROOT_PATH  = "${var.root_path}"
-      COMPONENTS = "${jsonencode(var.landing_zone_components)}"
+      ROOT_PATH  = var.root_path
+      COMPONENTS = jsonencode(var.landing_zone_components)
     }
   }
 }
 
 resource "null_resource" "landing_zone_apply" {
-  depends_on = ["null_resource.landing_zone_config"]
+  depends_on = [null_resource.landing_zone_config]
 
   triggers = {
-    command    = "${var.landing_zone_command}"
-    components = "${md5(jsonencode(var.landing_zone_components))}"
-    timestamp  = "${timestamp()}"
+    command    = var.landing_zone_command
+    components = md5(jsonencode(var.landing_zone_components))
+    timestamp  = timestamp()
   }
 
   provisioner "local-exec" {
     when    = "create"
     command = "node ${path.module}/scripts/apply.js"
 
     environment = {
-      ROOT_PATH  = "${var.root_path}"
-      COMMAND    = "${var.landing_zone_command}"
-      COMPONENTS = "${jsonencode(var.landing_zone_components)}"
+      OUTPUT_PATH = var.output_path
```

Comment:
please remove `output_path` as environment variable. update the internal scripts to use `~/.terrahub/cache/output/[project_code]/output.json` that can be used across modules ;)